python/taichi/lang/expr.py

```diff
@@ -1,8 +1,8 @@
-from .core import taichi_lang_core
-from .util import *
-from . import impl
-from .common_ops import TaichiOperations
-import traceback
+from taichi.lang import impl, runtime_ops
+from taichi.lang.common_ops import TaichiOperations
+from taichi.lang.core import taichi_lang_core
+from taichi.lang.util import (deprecated, is_taichi_class, python_scope,
+                              to_numpy_type, to_pytorch_type)
```

Comment:
I guess we need to reach a consensus on whether an absolute or an relative path should be used here :-) I guess using a relative path
1. is less verbose
2. makes it easier to refactor Python folder structure (without an IDE)

WDYT?

PyTorch seems to use the relative path approach: https://github.com/pytorch/pytorch/blob/98baad57642115d1f66723f6f10585ed933fd731/torch/fx/interpreter.py#L1-L6

Reply:
I'm afraid that I'd like to push back on this :-) For one thing, PEP8 recommends absolute imports (although not enforcing it). 

The major con of absolute import is its verbosity, which is usually a good thing from an engineering's perspective.  I think we need to start writing code in a clean and explicit manner, and stop trying to get away with things just because we can. (Consider what happens if we need to refer to another module that is a few layers above: `from ...foo.bar import baz`.) Furthermore, our package hierarchy isn't that deep, so the downside of abs import shouldn't be a problem most of the time. This style is also consistent with the CPP style where the include path always starts from the root directory.

Once all import issues are fixed, it should be much easier for us to switch to another style, compared to the current situation.

I don't think `pytorch` enforces their style well, as they also use absolute path in other places: https://github.com/pytorch/pytorch/blob/98baad57642115d1f66723f6f10585ed933fd731/torch/jit/_recursive.py#L10-L14. They also 

Sorry if I sound tired, cleaning up these imports is somewhat frustrating..

Reply:
Sounds good! I think your arguments totally make sense. If that is necessary, we can enforce absolute paths throughout the codebase, so that people do not need to further think/argue about this.

I understand that fixing issues like these can be boring and exhausting - thanks for all the hard work here!